### DISCUSSION TITLE
Release on Apple Silicon

### DISCUSSION THREAD

**Discussion by gongyu-lightmatter** [gitext-rs/git-stack#312] [Q&A] [answered]
Git stack has become an important tool for me. Recently, I started using the new Macbook Air and I am loving it. I was able to get git-stack working through `$ cargo install git-stack`.

However, it would be amazing if git-stack has a release for Apple Silicon!

**Comment by epage** [comment, marked as answer]
So far I've been modeling our pre-built binaries off of ripgrep which doesn't have macos arm support yet (BurntSushi/ripgrep#1737).  That issue does provide some hints though without a mac of my own, I'd be doing it all blind, having to do random releases to test each iteration.
